### PR TITLE
Update configuring-github-copilot-in-visual-studio.md

### PR DESCRIPTION
**Pull request by liorbp**
### Why:
Default shortcuts for VS are wrong

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Fix to show the right default shortcuts

### CODE REVIEW

**Comment by cmwilson21** [comment]
@liorbp Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by hubwriter** [comment]
@liorbp - Many thanks for raising this PR.

I'd like to hold off merging this PR just now while I investigate this a little. As a VS Code, but not Visual Studio, user, I believe the default keyboard shortcuts in Visual Studio:
* https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio#keyboard-shortcuts-for-github-copilot

should be the same as the shortcuts for VS Code:
* https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio-code?platform=windows#keyboard-shortcuts-for-github-copilot (Windows/Linux)
* https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio-code?platform=mac#keyboard-shortcuts-for-github-copilot (Mac)

If this is the case I think we should repeat the shortcut information currently used for VS Code on the page for Visual Studio.

File: content/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ You can use the default keyboard shortcuts in {% data variables.product.prodname
 
 | Action | Shortcut | Command name |
 |:---|:---|:---|
-|Show next inline suggestion|<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>]</kbd>|Tools.Nextsuggestion|
-|Show previous inline suggestion|<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>[</kbd>|Tools.Previoussuggestion|
+|Show next inline suggestion|<kbd>Alt</kbd>.<kbd>]</kbd>|Tools.Nextsuggestion|
```
Comment (hubwriter) [inline content/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio.md:26]:
```suggestion
|Show next inline suggestion|<kbd>Alt</kbd>+<kbd>.</kbd>|Tools.Nextsuggestion|
```

**Review by hubwriter — APPROVED** [review]
Many thanks @liorbp 👍 

I've confirmed the shortcuts (which are not the same as for VS Code). I've made a slight change: accepting the next suggestion is `Alt+.`

I'll get this merged.
Many thanks again for going out of your way to correct this. 
Much appreciated. 🥇

**Comment by liorbp** [comment]
@hubwriter Thanks for reviewing.
I agree the shortcut should be aligned between VS Code and Visual Studio.
Unfortunately this is not the case right now and the documentation is misleading.

File: content/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ You can use the default keyboard shortcuts in {% data variables.product.prodname
...
+|Show previous inline suggestion|<kbd>Alt</kbd>+<kbd>,</kbd>|Tools.Previoussuggestion|
```
Comment (shellsCryptoMarketing) [inline content/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio.md:27]: [drive1.txt](https://github.com/github/docs/files/10893356/drive1.txt)

**Review by shellsCryptoMarketing — COMMENTED** [review]
[xy2.txt](https://github.com/github/docs/files/10893366/xy2.txt)

**Comment by MahirAbdulatif42** [comment]
Opening contract

**Pull request merged**